### PR TITLE
Fix position source code files toggle

### LINKED ISSUE #60381: rustdoc: improve position and display of source code files sidebar
Screenshot 2019-04-29 at 23 16 34

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #60381.

The second commit is a big cleanup of the media queries.

r? @rust-lang/rustdoc 

cc @Manishearth 

screenshot of the fix:

Screenshot 2019-04-29 at 23 42 56

### CODE REVIEW

File: src/librustdoc/html/static/rustdoc.css [hunk 1]
```diff
@@ -1228,6 +1228,39 @@ h4 > .important-traits {
 	margin: 5px 0;
 }
 
+#sidebar-toggle {
+	position: fixed;
```
Comment (Manishearth) [inline src/librustdoc/html/static/rustdoc.css]: What was actually changed here? Moving the lines up makes it harder to see the actual diff.

File: src/librustdoc/html/static/rustdoc.css [hunk 2]
```diff
@@ -1261,7 +1138,177 @@ h4 > .important-traits {
 	margin-bottom: 6px;
 }
 
+.theme-picker {
+	position: absolute;
```
Comment (Manishearth) [inline src/librustdoc/html/static/rustdoc.css]: Does this commit just move entries around, or does it actually change any semantics too?

**Comment by Manishearth** [comment]
This fix seems to be pushing the line list back down (as compared to the screenshot in https://github.com/rust-lang/rust/issues/60381 ). Does this go away when https://github.com/rust-lang/rust/pull/60380 is applied?

(Also please explain in words what is being fixed in the issue or PR title, it's hard to guess this stuff at times)

File: src/librustdoc/html/static/rustdoc.css (hunk 1 above)
Comment (GuillaumeGomez) [reply src/librustdoc/html/static/rustdoc.css]: Just look at the first commit. I splitted the changed so you can review easily.

File: src/librustdoc/html/static/rustdoc.css (hunk 2 above)
Comment (GuillaumeGomez) [reply src/librustdoc/html/static/rustdoc.css]: The second commit is a cleanup.

**Comment by GuillaumeGomez** [comment]
> This fix seems to be pushing the line list back down (as compared to the screenshot in #60381 ). Does this go away when #60380 is applied?

In mobile version, it moves it down and make it bigger but on desktop, nothing changes.

**Comment by Dylan-DPC** [comment]
ping from triage @Manishearth waiting for your review on this

File: src/librustdoc/html/static/rustdoc.css (hunk 1 above)
Comment (Manishearth) [reply src/librustdoc/html/static/rustdoc.css]: I am looking at the first commit, it's still harder to tell what was added

Comment (Manishearth) [reply src/librustdoc/html/static/rustdoc.css]: Ah, the only change is adding the stuff to the media query.

**Pull request merged**